src/spec_tools/json_schema.cljc

```diff
@@ -21,21 +21,33 @@
 (defn- spec-dispatch [dispatch spec children] dispatch)
 (defmulti accept-spec spec-dispatch :default ::default)
 
+;;
 ;; predicate list taken from https://github.com/clojure/clojure/blob/master/src/clj/clojure/spec/gen.clj
+;;
 
 ; any? (one-of [(return nil) (any-printable)])
+(defmethod accept-spec 'clojure.core/any? [_ _ _] {:type "object"})
+
 ; some? (such-that some? (any-printable))
+(defmethod accept-spec 'clojure.core/some? [_ _ _] {:type "object"})
+
 ; number? (one-of [(large-integer) (double)])
+(defmethod accept-spec 'clojure.core/number? [_ _ _] {:type "double"})
```

Comment:
Should be `"number"` as there's no `double` in JSON-Schema; in Swagger schema you could have `:format "double"`. Applies to `double`s below as well.